doc/classes/Material.xml

```diff
@@ -12,8 +12,10 @@
 	</methods>
 	<members>
 		<member name="next_pass" type="Material" setter="set_next_pass" getter="get_next_pass">
+			Sets the [Material] to be used for the next pass. This renders the object again using a different material. [b]Note:[/b] only applies to [SpatialMaterial]s and [ShaderMaterial]s with type "Spatial".
```

Comment:
I'd through the `[b]Note:[/b]` on its own line as done for most other occurrences (same for all added notes in this patch).